src/compute-client/src/controller/instance.rs

```diff
@@ -1023,20 +1009,21 @@ where
         otel_ctx: OpenTelemetryContext,
         replica_id: ReplicaId,
     ) -> Option<ComputeControllerResponse<T>> {
-        let peek = match self.compute.peeks.get_mut(&uuid) {
-            Some(peek) => peek,
-            None => {
-                tracing::warn!("did not find pending peek for {}", uuid);
-                return None;
-            }
-        };
-
         // Forward the peek response, if we didn't already forward a response
         // to this peek previously. If the peek is targeting a replica, only
         // forward the response from that replica.
         // TODO: we could collect the other responses to assert equivalence?
```

Comment:
I think also this TODO can go too. Now that we allow to compact away data slow replicas might still use, we can't wait for all and compare anymore as the late ones might produce errors.

Reply:
Yes, that makes sense!